sdk/core/core-lro/test/utils/testServiceClient.ts

```diff
@@ -19,7 +21,7 @@ export class TestServiceClient extends ServiceClient {
 
   constructor(
     credentials: TokenCredential | ServiceClientCredentials,
-    options?: ServiceClientOptions
+    options?: TestServiceClientOptions
```

Comment:
This change follows the updated rule here: https://github.com/Azure/azure-sdk-for-js/pull/11403.